src/app/app.component.scss

```diff
@@ -1,10 +1,10 @@
-@import url(https://overpass-30e2.kxcdn.com/overpass.css);
+//@import url(https://overpass-30e2.kxcdn.com/overpass.css);
 
 .app {
   padding-top: 20px;
   flex: 1;
-  font-family: 'overpass';
-  font-size: 11pt;
+  // font-family: 'overpass';
```

Comment:
Why commenting it out the font and the size? Afaik we have approved that with @ALRubinger during the pre-summit review meeting